### PR TITLE
test new changes to open3d slam

### PR DESCRIPTION
**Pull request by deepanaishtaweera**

### CODE REVIEW

File: smb_slam/launch/localization.launch
Comment (mantelt) [inline smb_slam/launch/localization.launch:1]: the purpose of this file used to be to localize in an existing point cloud. is that functionality not available anymore? Or do I miss something? 

(asking due to removal of argument `pcd_file_path`)

Comment (deepanaishtaweera) [reply smb_slam/launch/localization.launch:1]: pcd file path is not an argument anymore in the launch file, but it is a paramater in the lua file, there's a fix that needs to be done for the localisation to work which I have informed Turcan and he's working on it.

if the pcd_file_path needs to be an argument in the launch file, i will inform it.

File: smb_slam/launch/localization.launch [hunk 1]
```diff
@@ -5,24 +5,25 @@
   <arg name="use_sim_time"          default="false" />
   <arg name="cloud_topic"           default="/rslidar/points"  doc="input topic for the lidar pointcloud"/>
   <arg name="parameter_filename"    default="param_rs16_localization.lua" doc="param file name"/>
```
Comment (mantelt) [inline smb_slam/launch/localization.launch:7]: please extend the doc string to point out that the pcd file is specified in the lua file.

**Review by mantelt — APPROVED** [review]
thanks, lgtm!

**Pull request merged**